web/src/components/nav/betaNotice.js

```diff
@@ -0,0 +1,20 @@
+import { Route } from "react-router-dom";
+import { DismissableMessage } from "../ui";
+
+const MESSAGESTYLE = {
+	background: "#fec61e",
+	marginTop: "1em",
+	display: "inline-block",
+	width: "100%",
+};
+
+const MESSAGECONTENT =
```

Comment:
This should be written in camel case.

Reply:
As it's a constant that is scoped to at least the whole file, I can see the argument for non-camelcase. Although, I would advocate for SCREAMING_SNAKE_CASE.

```suggestion
const MESSAGE_CONTENT =
```